src/lib/es2015.iterable.d.ts

```diff
@@ -57,18 +57,18 @@ interface Array<T> {
 
 interface ArrayConstructor {
     /**
-     * Creates an array from an iterable object.
-     * @param iterable An iterable object to convert to an array.
+     * Creates an array from an array-like or iterable object.

```

Comment:
is 'array-like' an ecmascript concept? If it's just in typescript type system, I think it would be better not to reference it as separate from iterables.

Reply:
Per [the spec](https://tc39.es/ecma262/#sec-lengthofarraylike), an array-like object is anything with a number length property. [The `Array.from` section](https://tc39.es/ecma262/#sec-array.from) mentions it too.